pmoves/data/consciousness/load_supabase_chunks.py

```
@@ -53,11 +66,12 @@ def main():
 
     print(f"Wrote {len(sql_lines)} SQL statements to {temp_sql}")
 
-    # Load via Docker
+    pgpassword = _db_password()
+
+    # Load via Docker (password injected via `-e`, never inlined in a shell string)
     result = subprocess.run([
```

Ruff (0.15.17)

[error] `subprocess` call: check for execution of untrusted input

(S603)